protosocket-rpc/src/client/rpc_client.rs

```diff
@@ -42,6 +42,13 @@ where
         }
     }
 
+    /// Check if the client is still connected to the server.
+    ///
+    /// We need this is_alive flag to be public so that the client can check if the connection is still alive.
```

Comment:
```suggestion
    /// Checking this before using the client does not guarantee that the client is still alive when you send
    /// your message. It may be useful for connection pool implementations - for example, [bb8::ManageConnection](https://github.com/djc/bb8/blob/09a043c001b3c15514d9f03991cfc87f7118a000/bb8/src/api.rs#L383-L384)'s
    /// is_valid and has_broken could be bound to this function to help the pool cycle out broken connections.
```

This line is a PR comment. Documentation strings should be relevant over time!